File: In_C_Programming/q7.c
#include <stdio.h>
int count_Min(int arr[], int ln)
{

    int min = arr[0];

    for (int i = 0; i < ln; i++)
    {
        if (arr[i] < min)
        {
            min = arr[i];
        }
    }

    return min;
}

int main()
{

    int size;

    printf("Enter THe Array Length: \n");
    scanf("%d", &size);

    int arr[size];

    printf("Enter The Elements: \n");

    for (int i = 0; i < size; i++)
    {
        scanf("%d", &arr[i]);
    }

    int result = count_Min(arr, size);

    printf("The Max element of the Array is: %d \n", result);

    return 0;
}

File: In_Java_Programming/q7.java
import java.util.Scanner;

public class q7 {

    public static int getMin(int arr[], int size) {

        int min = arr[0];

        for (int i = 0; i < size; i++) {

            if (arr[i] < min) {

                min = arr[i];
            }
        }

        return min;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        int x[] = new int[ln];

        int result = getMin(x, ln);

        System.out.println("The Max Element is: " + result);

        sc.close();
    }
}
